index.src.html

```diff
@@ -713,16 +713,17 @@ <h3 id="determine-requests-referrer">
   <code>referrer</code> will be <code>no referrer</code>, and Fetch won't call
   into this algorithm.
 
-  ISSUE: This algorithm needs to be modifed to take into account any local
-  overrides via the referrer attribute on elements.
-
   <ol>
     <li>
       Let <var>environment</var> be <var>request</var>'s <code>client</code>.
     </li>
     <li>
-      Let <var>policy</var> be the value of <var>environment</var>'s
-      <a>referrer policy</a>.
+      If <var>request</var> was initiated by an element with
+      a <code>referrerpolicy</code> attribute, let <var>policy</var> be
+      the result of executing [[#determine-policy-for-token]] on the
```

Comment:
Ah, right, so this should just be "If request was initiated by an element with a referrerpolicy content attribute, let policy be the attribute's value. Otherwise..."?